.github/workflows/release.yaml

```diff
@@ -16,10 +16,12 @@ jobs:
           for f in $(ls *bundle.yaml); do
             cp ${f} ${f/\.yaml/-${VERSION}\.yaml}
           done
-          
```

Comment:
FWIW this is just `pre-commit` cleaning up whitespace that `shellcheck` somehow missed.